### ISSUE TITLE
Status of the project?

### ISSUE THREAD

**Issue by alanjosephwilliams** [SnapFresh/SnapFresh#45]
Hey folks—

Curious if you could update the Code for America set about the status of the SnapFresh project—both this iOS version, as well as the webapp (could you point me to that repo?).

There is a group of us working for CfA that are focused largely on low-income citizens enrolled in SNAP and Medicaid. Here are some of our nascent projects:

Enrollment:
https://github.com/codeforamerica/calfresh-and-so-clean

Check your EBT balance via SMS:
https://github.com/codeforamerica/balance

Don't wait on hold when you call HSA:
https://github.com/codeforamerica/connect

Find EBT retailers, and coming soon, ATMs where there is no fee:
https://github.com/fureigh/ebt-near-me

Obviously, we are interested getting a better understanding the status and future of SnapFresh because of that last one.
